autopush/web/webpush.py

```diff
@@ -477,6 +477,9 @@ def post(self,
         encoding = ''
         if notification.data and notification.headers:
             encoding = notification.headers.get('encoding', '')
+            self.metrics.increment(
```

Comment:
When there's no body we'll get an empty string and thus a "updates.notification.encoding." metric which is a bit odd, but I don't actually care :)